README.md

```diff
@@ -20,34 +20,39 @@ Variables without a default value and not marked optional, *MUST* be defined for
 | GP2GP_LOGGING_FORMAT               | (*)                       | Defines how to format log events on stdout
 
 
-(*) SCR API is using logback (http://logback.qos.ch/) for logging configuration.
+(*) GP2GP API is using logback (http://logback.qos.ch/) for logging configuration.
 Default log format is defined in the built-in logback.xml (https://github.com/NHSDigital/summary-care-record-api/tree/master/docker/service/src/main/resources/logback.xml)
-This value can be overriden using `SCR_LOGGING_FORMAT` environment variable.
+This value can be overriden using `GP2GP_LOGGING_FORMAT` environment variable.
 Alternatively, an external `logback.xml` with much more customizations can be provided using `-Dlogback.configurationFile` JVM parameter.
 
 ## How to run service:
-TODO
-<!-- * Navigate to `ddocker/service`
+* Navigate to `docker/service`
 * Add environment tag `export TAG=latest`
 * Run script: `build-image.sh` (excute privileges might be required `chmod +x build-image.sh`)
```

Comment:
Should this be `build-service-image.sh`?

Reply:
Could probably just skip this step - `start-local-environment.sh` calls this script anyway